src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -105,20 +105,17 @@
     <Copy SourceFiles="@(ILLinkAnalyzersAssemblies)" DestinationFiles="@(ILLinkAnalyzersAssemblies->'$(AnalyzerAssembliesDirectory)/%(RecursiveDir)%(Filename)%(Extension)')" />
   </Target>
 
-  <Target Name="PublishDotnetFormat"
-          BeforeTargets="Build;PublishNetSdk">
+  <Target Name="PublishDotnetFormat" AfterTargets="Build" BeforeTargets="PublishNetSdk">
     <PropertyGroup>
       <DotnetFormatDestinationDirectory>$(OutputPath)/DotnetTools/dotnet-format</DotnetFormatDestinationDirectory>
     </PropertyGroup>
 
     <ItemGroup>
-      <DotnetFormatDllFiles Include="$(Pkgdotnet-format)/tools/net9.0/any/**/*.dll" />
-      <DotnetFormatConfigFiles Include="$(Pkgdotnet-format)/tools/net9.0/any/**/*.json" />
-      <DotnetFormatConfigFiles Include="$(Pkgdotnet-format)/tools/net9.0/any/**/*.config" />
+      <DotnetFormatDllFiles Include="$(ArtifactsBinDir)/dotnet-format/$(Configuration)//$(NetCurrent)/**/*.dll" />
```

Comment:
I changed this to use the output directly to find dotnet-format dlls.

Reply:
This means we should be able to remove the sdk dependency on the dotnet-format package.